pkg/auth/authenticator/token/apitoken/token.go

```diff
@@ -0,0 +1,47 @@
+package apitoken
```

Comment:
> critics have to provide suggestions

`remotemaster` ?  I already know its a token, but I'd like to know which kind of token it is.